app/src/main/res/values/strings.xml

```diff
@@ -5,8 +5,8 @@
     <string name="action_search">Search</string>
     <string name="action_add">Add</string>
 
-    <string name="noGiftCards">Click the "+" (plus) button to add a card first.\n\nCatima carries your cards on the device, so they are always within reach.</string>
-    <string name="noMatchingGiftCards">Didn\'t find anything. Try changing your search.</string>
+    <string name="noGiftCards">Click the "+" (plus) button to add a card first.</string>
+    <string name="noMatchingGiftCards">Change the search to try again.</string>
```

Comment:
That really makes it too little info for the user to understand what is wrong. Again on an already very empty screen:

![image](https://user-images.githubusercontent.com/1885159/113521449-cace8880-9599-11eb-8cb9-2975d38e14b5.png)

Reply:
The original still leaves the causal effect up in the air, but longer and more complex at that.

Reply:
"Didn't find anything" states that a search was done. So your statement is an argument in favour of my original version.

Reply:
I couldn't really think of anything, so I reverted it.